### DISCUSSION TITLE
InfluxDB update frequency

### DISCUSSION THREAD

**Discussion by andygodber** [jertel/vuegraf#53] [Ideas]
Are there any issues with changing the influx update frequency (default  set at 60s) ?
I tried this on the Python code and let it run for a while with no discernible impact, even down to 5s.

If not, would it be possible to make it parameterised?

And, not being a Docker expert (other than following instructions), how could I make the change in that version?

**Comment by jertel** [comment]
Yes, it's possible, but consider that this Emporia API is an unpublished API that we've tapped into. If Vuegraf users start adjusting this value to very short intervals and cause the load on Emporia's servers to spike, then the Vuegraf community as a whole will be the ones having to deal with the fallout of new API changes, or stricter security controls. So I ask that you please consider whether you actually need rapid updates at that 10s frequency. What are you actually getting out of it?

I have pushed a change to allow this setting to be customized in the vuegraf.json file. Please use it wisely.

```
...
    },
    "updateIntervalSecs": 30,
    "accounts": [
...
```

You will need to run `docker pull jertel/vuegraf` to grab the new Docker image.

**Comment by andygodber** [comment]
Thank you for the update - seems to be working well.
Noted and accepted on not abusing the frequency of API use.
